### PR TITLE
DOCSUP-5634: REPLACE TABLE

### PR DESCRIPTION
**Pull request by adevyatova**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #18521

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

...

Detailed description / Documentation draft:

...

By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

File: docs/en/engines/database-engines/atomic.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+
+# Atomic {#atomic}
+
+It is an experimental database engine, it supports non-blocking `DROP` and `RENAME TABLE` queries and atomic `EXCHANGE TABLES t1 AND t2` queries.
+
+## Creating a Database {#creating-a-database}
+
+    CREATE DATABASE test ENGINE = Atomic;
```
Comment (olgarev) [inline docs/en/engines/database-engines/atomic.md:13]:
```suggestion
``` sql
    CREATE DATABASE test ENGINE = Atomic
```    
```

File: docs/en/sql-reference/statements/create/table.md [hunk 2]
```diff
@@ -259,3 +259,68 @@ CREATE TEMPORARY TABLE [IF NOT EXISTS] table_name
...
+
+**Examples:**
+
+`REPLACE` for a non-existent table will cause an error.
+
+```sql
+CREATE DATABASE base ENGINE = Atomic;
+REPLACE TABLE base.t1 (n UInt64, s String) ENGINE = MergeTree ORDER BY n; -- { serverError 60 }
+```
+
```
Comment (olgarev) [inline docs/en/sql-reference/statements/create/table.md:303]: Example is not necessary here, typically we don't give examples with errors in documentation.

**Comment by alexey-milovidov** [comment]
Atomic database engine is production grade and is used by default.
Please remove the mentions that it's experimental from the docs.

**Comment by adevyatova** [comment]
> Atomic database engine is production grade and is used by default.
> Please remove the mentions that it's experimental from the docs.

Fix it. Atomic database engine will be described in another PR.

**Review by tavplubix — APPROVED** [review]

**Pull request merged**